packages/graph-engine/src/nodes/color/create.ts

```diff
@@ -95,25 +95,32 @@ export default class NodeDefinition extends Node {
       },
     });
     this.addInput("a", {
-      type: NumberSchema,
-      visible: true,
+      type: {
+        ...NumberSchema,
+        default: "255",
```

Comment:
Is this always correct? I'm pretty sure a lot of spaces have 0-1 inputs. Lets rather not define inputs if they're going to be incorrect

Reply:
lets go with 0